jloadr-bootstrap/src/main/java/de/adito/jloadr/bootstrap/DigestingInputStream.java

```diff
@@ -50,7 +50,7 @@ public static MessageDigest getMessageDigest()
     }
     catch (NoSuchAlgorithmException pE)
     {
-      throw new RuntimeException();
+      throw new RuntimeException("Was not able to digest message.");
```

Comment:
Do you really want to omit the stacktrace of "pE" here?

Reply:
Oops. Good catch.